templates/guides/fastly_object_storage.md

```diff
@@ -0,0 +1,117 @@
+---
+page_title: Managing content in Fastly Object Storage
+subcategory: "Guides"
+---
+# Managing content in Fastly Object Storage
+
+Content in Fastly Object Storage (buckets and objects) can be managed
+using a combination of this provider (Fastly) and the Amazon Web
+Services (AWS) provider, since Fastly Object Storage provides an AWS
+S3-compatible API.
+
+## Example Configuration
+
+Two HCL files are required. The first, `main.tf`:
+
+```terraform
+terraform {
+  required_providers {
+    fastly = {
+      source  = "fastly/fastly"
+      version = "~> 7.1.0"
+    }
+  }
+}
+
+resource "fastly_object_storage_access_keys" "main" {
+  description = "FOS Key"
+  permission  = "read-write-admin"
+}
+
+module "fos" {
+  source = "./fos"
+  access_key_id = fastly_object_storage_access_keys.main.id
+  secret_key    = fastly_object_storage_access_keys.main.secret_key
+}
+```
```

Comment:
It feels like there's something missing here, either a missing word (which requires correcting punctuation, or some sort of followup statement to complete the second sentence (which requires text after the code block). The `fos/main.tf` code block below this states that it need to be placed and named specifically. Perhaps that was the plan for `main.tf` as well?